swiggy-pdp-cart/src/banner.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import momos from "./images/momos.jpg"
+
+export default function Banner(){
+    return(
+        <div className="banner">
+            <img src={momos} alt="Momos"/>
+            <h1>Punjabi Momos</h1>
```

Comment:
Make this configurable and prop driven so that it can be used at other places as well